browser.js

```diff
@@ -4,6 +4,7 @@ function oldBrowser () {
   throw new Error('secure random number generation not supported by this browser\nuse chrome, FireFox or Internet Explorer 11')
 }
 
+var Buffer = require('safe-buffer')
```

Comment:
Fixed to be `require('safe-buffer')` in https://github.com/crypto-browserify/randombytes/commit/1870ca394623e1c88b8188a45a92a2dd7b0d5c53